pkg/api/preferences.go

```diff
@@ -34,13 +34,9 @@ func (hs *HTTPServer) SetHomeDashboard(c *models.ReqContext) response.Response {
 
 // GET /api/user/preferences
 func (hs *HTTPServer) GetUserPreferences(c *models.ReqContext) response.Response {
-	return hs.getPreferencesFor(c.Req.Context(), c.OrgId, c.UserId, 0)
-}
-
-func (hs *HTTPServer) getPreferencesFor(ctx context.Context, orgID, userID, teamID int64) response.Response {
-	prefsQuery := models.GetPreferencesQuery{UserId: userID, OrgId: orgID, TeamId: teamID}
+	prefsQuery := models.GetPreferencesQuery{UserId: c.UserId, OrgId: c.OrgId, TeamId: 0}
 
-	if err := hs.SQLStore.GetPreferences(ctx, &prefsQuery); err != nil {
+	if err := hs.PreferencesService.GetPreferences(c.Req.Context(), &prefsQuery); err != nil {
```

Comment:
As we no longer follow the bus patterns - this can be changed to _return_ results, `if result, err := ....`.

Reply:
yes, that's the plan